pkg/discoverymanager/discovery.go

```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/liqotech/liqo/pkg/clusterid"
 )
 
+// clusterRole
+// +kubebuilder:rbac:groups=discovery.liqo.io,resources=foreignclusters,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=discovery.liqo.io,resources=foreignclusters/status,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=core,resources=nodes,verbs=list;watch
```

Comment:
```suggestion
// +kubebuilder:rbac:groups=core,resources=nodes,verbs=get;list;watch
```

Just to avoid possible unwanted issues.